arroyo/examples/transform_and_produce/script.py

```diff
@@ -0,0 +1,53 @@
+from arroyo.backends.kafka.configuration import build_kafka_consumer_configuration
+from arroyo.backends.kafka.consumer import (
+    KafkaConsumer,
+    KafkaProducer,
+)
+from arroyo.processing.processor import StreamProcessor
+
+
+from arroyo.types import Topic
+import socket
+
+from arroyo.examples.transform_and_produce.factory import (
+    HashPasswordAndProduceStrategyFactory,
+)
+
+
+RAW_TOPIC = Topic("raw-topic")
+HASHED_TOPIC = Topic("hashed-topic")
+
+
+if __name__ == "__main__":
+
+    # Create a Producer, KafkaProducer is a wrapper on confluent_kafka.Producer
+    producer = KafkaProducer(
+        {
+            "bootstrap.servers": "localhost:9092",  # Where the broker is, localhost:9092 when running locally
```

Comment:
Can you define it as a variable at the top of the file, and maybe allow it to be over-ridden via an environment variable or something?